rust-atuin.spec

```diff
@@ -0,0 +1,82 @@
+# Generated by rust2rpm 25
+
+# prevent library files from being installed
+%global __cargo_is_lib() 0
+
+%global crate atuin
+
+Name:           rust-atuin
+Version:        18.0.2
+Release:        %autorelease
+Summary:        Magical shell history
+
+License:        MIT
+URL:            https://crates.io/crates/atuin
+Source:         %{crates_source}
+Source:         atuin-%{version}-vendor.tar.xz
+
+BuildRequires:  cargo-rpm-macros >= 25
+# Testing dependencies
+BuildRequires:  postgresql
+
+%global _description %{expand:
+Atuin replaces your existing shell history with a SQLite database, and records
+additional context for your commands. Additionally, it provides optional and
+fully encrypted synchronisation of your history between machines, via an Atuin
+server.}
+
+%description %{_description}
+
+%package     -n %{crate}
+Summary:        %{summary}
+
+%description -n %{crate} %{_description}
+
+
+%prep
+%autosetup -n %{crate}-%{version} -p1 -a1
+%cargo_prep -v vendor
+
+
+%build
+%cargo_build
+%{cargo_license_summary}
+%{cargo_license} > LICENSE.dependencies
+%{cargo_vendor_manifest}
+
+
+%install
+%cargo_install
+
+# Generate all of the shell-completions
+for completion in bash fish zsh; do
+  %{buildroot}%{_bindir}/atuin gen-completions --shell $completion -o .
+done
+install -Dpm 644 atuin.bash %{buildroot}%{_datadir}/bash-completion/completions/atuin
+install -Dpm 644 atuin.fish %{buildroot}%{_datadir}/fish/completions/atuin
+install -Dpm 644 _atuin %{buildroot}%{_datadir}/zsh/site-functions/atuin
+
+# Add atuin to default profile
+mkdir -p %{buildroot}%{_sysconfdir}/profile.d
+cat > %{buildroot}%{_sysconfdir}/profile.d/atuin.sh <<EOF
+$(%{buildroot}%{_bindir}/atuin init bash)
+EOF
+
+
+%check
+%cargo_test
```

Comment:
```suggestion
%cargo_test -- -- --skip sync --skip change_password --skip multi_user_test --skip registration
```

For now, let's skip the tests that require external integration. (syntax from [here](https://pagure.io/fedora-rust/rust-packaging/blob/main/f/macros.d/macros.cargo#_163), I think I got it right?)